src/pages/AddNewWorker/index.tsx

```diff
@@ -1,11 +1,53 @@
-import React from 'react'
+import React, { useState, useContext } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { AppContext } from '../../Context/AppContext';
+import { Employee } from '../../Context/hooks/useAppContext';
+import './AddNewWorker.scss';
+
+const AddNewWorker: React.FC = () => {
+  const { employees, setEmployees } = useContext(AppContext);
+  const [employee, setEmployee] = useState<Employee>({
+    id: employees.length + 1,
+    firstName: '',
+    lastName: '',
+    birthDate: '',
+    address: {
+      address: '',
+      city: '',
+    },
+    phone: '',
+    email: '',
+    image: '',
+  });
+  const navigate = useNavigate();
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>) => {
+    const { name, value } = e.target;
+    setEmployee({ ...employee, [name]: value });
+  };
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    setEmployees([...employees, employee]);
+    navigate('/user-list');
+  };
 
-const AddNewWorker = () => {
   return (
-    <div>
-      
+    <div className="add-worker">
+      <h1 className="add-worker__title">Dodaj Pracownika</h1>
```

Comment:
h1 should be one per page, please use h2

Reply:
done